Modules/Server/Sources/PocketCastsServer/Public/Sync/SyncManager.swift

```diff
@@ -61,5 +61,6 @@ public class SyncManager {
         KeychainHelper.removeKey(ServerConstants.Values.syncingEmailKey)
         KeychainHelper.removeKey(ServerConstants.Values.syncingPasswordKey)
         KeychainHelper.removeKey(ServerConstants.Values.syncingV2TokenKey)
+        KeychainHelper.removeKey(ServerConstants.Values.appleAuthIdentityTokenKey)
```

Comment:
@leandroalonso I just noticed this fails to remove the `appleAuthUserIDKey` so I'll push that change after your review